src/api/flaskr/service/learn/funcs.py

```
@@ -353,7 +353,9 @@ def get_study_record(
             )
             for i in attend_scripts
         ]
-        user_info = User.query.filter_by(user_id=user_id).first()
+        user_info = load_user_aggregate(user_id)
+        if not user_info:
+            raise_error("USER.USER_NOT_FOUND")
```

Ruff (0.14.1)

Undefined name `raise_error`

(F821)